docs/install/containers.md

```diff
@@ -77,6 +77,22 @@ docker run --rm \
   --output harvest-compose.yml
 ```
 
+By default, the above command uses the harvest configuration file located in the current directory, denoted as `$(pwd)/harvest.yml`. However, you may want to use a harvest config from a different location.
+??? question "What if my harvest configuration file is in a different path"
```

Comment:
??? question "What if my harvest configuration file is somewhere else or not named harvest.yml"

Use the following `docker run` command, updating the `HYML` variable to point to your `harvest.yml`